CodeEdit/NavigatorSidebar/FindNavigator/FindNavigatorModeSelector.swift

```diff
@@ -82,7 +82,7 @@ struct FindNavigatorModeSelector: View {
                 } label: {
                     HStack(spacing: 2) {
                         if index > 0 {
-                            chevron
+                            // chevron
```

Comment:
make sure to remove all commented code. [Reference](https://github.com/CodeEditApp/CodeEdit/wiki/Code-Style#remove-comments-and-print)